paddle/cinn/runtime/flags.cc

```diff
@@ -160,6 +160,11 @@ DEFINE_int32(cinn_profiler_state,
              "Specify the ProfilerState by Int in CINN, 0 for kDisabled, 1 for "
              "kCPU, 2 for kCUDA, 3 for kAll, default 0.");
 
+DEFINE_int32(cinn_schedule_error_message_level,
```

Comment:
Same as my other comment, can the ErrorMessage be use at other places? If so, should we remove "schedule" text in the flag? Then we can use the code you implemented all over the CINN!